calcom/cal.com
# V2 Settings Shell - Add Teams Section & UI fixes

## What does this PR do?



This PR adds teams pages to the settings shell. This PR also contains UI fixes.

Routing to the specific team pages will be added once those pages are completed

![image](https://user-images.githubusercontent.com/65426560/189425574-a7fc1b77-6370-4466-b2bc-945899b7d01c.png)


Fixes #4105



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Sign in using the teampro user
- Go to the settings page
- The Seeded Team should appear on the side and all 5 team pages should be present
- Sign into the teampro2 user
- Go to the settings page
- Only the profile and members pages should be available

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
